presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraPartitionManager.java

```diff
@@ -114,35 +114,31 @@ public CassandraPartitionResult getPartitions(ConnectorTableHandle tableHandle,
             return ImmutableList.of();
```

Comment:
About commit message:
- `Add doc for CassandraSession#getPartitions` there is no point to mention such minor things
- `Add a legacy parttion fetcher for cassandra version < 2.2`, instead I would say that old method to retrieve partition list is still used for cassandra version < 2.2 where IN predicate is not supported.
- you are missing the most important part, the motivation and outcome. Please write something why this change is introduced and what is a desired new behaviour.

Reply:
@kokosing how's f753c56 ? ;)